<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FriendDialog</class>
 <widget class="QDialog" name="FriendDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(236, 157, 0);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>客官给钱</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>客官给钱的签名</string>
   </property>
  </widget>
  <widget class="QLabel" name="photoLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	image: url(:/images/touxiang01.jpg);
  background-color:green;
  border-radius:4px;
}
QLabel:hover{
   border:2px solid rgb(12, 255, 235);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="MyLabel" name="myLabel_3">
      <property name="styleSheet">
       <string notr="true">QLabel{
image: url(:/images/icon_contacts_normal.png);
}
QLabel:hover{
image: url(:/images/icon_contacts_selected.png);
}</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="MyLabel" name="myLabel_4">
      <property name="styleSheet">
       <string notr="true">QLabel{
image: url(:/images/icon_group_normal.png);
}
QLabel:hover{
image: url(:/images/icon_group_selected.png);
}</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="MyLabel" name="myLabel_5">
      <property name="styleSheet">
       <string notr="true">QLabel{
image: url(:/images/icon_last_normal.png);
}
QLabel:hover{
image: url(:/images/icon_last_selected.png);
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="MyLabel" name="myLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
  
   color:white;
}
QLabel:hover{
  background-color:rgb(212, 64, 39);
  color:rgb(255, 255, 255);
  border-top-right-radius:4px;
}</string>
   </property>
   <property name="text">
    <string>—</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="MyLabel" name="myLabel_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
  
   color:white;
}
QLabel:hover{
  background-color:rgb(212, 64, 39);
  color:rgb(255, 255, 255);
  border-top-right-radius:4px;
}</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_addContact">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="MyLabel" name="searchButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>590</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>查找</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>199</y>
     <width>251</width>
     <height>381</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page1">
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>251</width>
       <height>381</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="friendAreaLayout">
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>231</width>
       <height>371</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="groupAreaLayout">
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3"/>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header>mylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
